core/src/main/java/cz/o2/proxima/storage/internal/AbstractDataAccessorFactory.java

```diff
@@ -69,6 +70,29 @@ default void setup(Repository repo) {}
    * @param uri the URI to create accessor for
    * @param cfg optional additional configuration
    * @return {@link AbstractDataAccessor} for given operator and family
+   * @deprecated replaced with {@link AbstractDataAccessorFactory#createAccessor(DataOperator,
+   *     AttributeFamilyDescriptor)}
    */
-  T createAccessor(OP operator, EntityDescriptor entity, URI uri, Map<String, Object> cfg);
+  @Deprecated
+  default T createAccessor(OP operator, EntityDescriptor entity, URI uri, Map<String, Object> cfg) {
```

Comment:
I think we are free to do this breaking change. This tends to do more harm than good, over time.

Reply:
That will require a lot of changes in existed accessors which I'll do later in https://github.com/O2-Czech-Republic/proxima-platform/issues/192